app/src/controllers/initialData/sagas.js

```diff
@@ -41,12 +41,12 @@ function* fetchInitialData() {
   yield put(fetchUserAction());
   const userResult = yield take([FETCH_USER_SUCCESS, FETCH_USER_ERROR]);
   if (!userResult.error) {
+    yield put(authSuccessAction());
     const { payload: activeProjectKey } = yield take(SET_ACTIVE_PROJECT_KEY);
     yield put(fetchProjectAction(activeProjectKey));
     yield take(FETCH_PROJECT_SUCCESS);
     yield put(fetchPluginsAction());
     yield put(fetchGlobalIntegrationsAction());
-    yield put(authSuccessAction());
```

Comment:
What is the reason of moving it upper?
It was triggered as success action after fetching project to unblock further navigation and app loading once the project exists.

Reply:
Now there are cases when there is no project.
If there is no projectKey, then without authSuccessAction then nothing will work.